coder_sniffer/DrupalPractice/Sniffs/General/OptionsTSniff.php

```diff
@@ -81,6 +81,21 @@ public function process(File $phpcsFile, $stackPtr)
             $statementEnd = $tokens[$arrayToken]['bracket_closer'];
         }
 
+        // We want to find if the element "#options" belongs to a form element.
+        // Array with selectable options for a form element.
+        $formElements = [
+          "'checkboxes'",
+          "'radios'",
+          "'select'",
+          "'tableselect'",
+        ];
```

Comment:
Coder uses 4 space based indenting, this array currently uses 2 space based indenting.

Reply:
Oops, missed it. My bad